### PR TITLE
Update roadmap.yaml

### PR DESCRIPTION
**Pull request by srgregory-po**
Updating the FY21 roadmap for non-services work.  Added in WaterAlert and references to our latest product plan.

Before making a pull request
----------------------------

- [ ] Update the changelog appropriately

Title
-----------
FY21 update to public roadmap (covers all but service tasks)

Description
-----------
Updating the public roadmap with the latest product plans for FY21.  These changes cover the updates needed for the non-services work.  Assume services update to this roadmap is needed as a separate pull request.  No associated JIRA ticket, PO task.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: data/roadmap/roadmap.yaml [hunk 1]
```diff
@@ -48,133 +48,92 @@ categories:
...
-      -
-        status: Done
-        thing: "18F Phase I: Roadmap & Low-fidelity prototyping"
+        status: on deck
+        thing: "Search and Filter mapper beta development"
       -
-        status: Done
-        thing: "18F Phase II: Planning"
+        status: on deck
+        thing: "Consolidated download beta development"
```
Comment (chopkins-usgs) [inline data/roadmap/roadmap.yaml:60]: Did we ever do an alpha development?

File: data/roadmap/roadmap.yaml [hunk 2]
```diff
@@ -48,133 +48,92 @@ categories:
...
-        status: "In progress"
-        thing: "18F Phase II: Iterate"
-      -
-        status: "On deck"
-        thing: "18F Phase II: Transition & Handoff"
+        status: upcoming
+        thing: "Decommissioning NWISmapper"
   -
     name: "Next-generation monitoring location pages"
     status: "in progress"
```
Comment (chopkins-usgs) [inline data/roadmap/roadmap.yaml:66]: When do we consider these "done"?

File: data/roadmap/roadmap.yaml [hunk 3]
```diff
@@ -48,133 +48,92 @@ categories:
...
-        status: in progress
-        thing: "Implement new date range picker and tooltips"
-      - status: in progress
-        thing: "Use USGS National Map background layers"
-      - status: done
-        thing: "Put AHPS flood levels on gage height hydrograph"
       -
         status: done
-        thing: "Compile user feedback and prioritize new features and usability improvements"
+        thing: "Site-specific alerts (info, warning, errors)"
```
Comment (chopkins-usgs) [inline data/roadmap/roadmap.yaml:70]: Is this Water Alert?

File: data/roadmap/roadmap.yaml [hunk 4]
```diff
@@ -48,133 +48,92 @@ categories:
...
       -
-        status: Done
-        thing: "NWISweb user research (18F Phase I)"
+        status: on deck
+        thing: "Discovery on modernizing WaterQualityWatch, WaterWatch, GroundWaterWatch"
       -
-        status: "In Progress"
-        thing: "WDFN user testing (18F Phase II)"
+        status: upcoming
+        thing: "Discovery on delivering multiple sites simultaneously"
```
Comment (chopkins-usgs) [inline data/roadmap/roadmap.yaml:132]: Also in discovery ( on roadmap): Services and Documentation, and Reference Lists (also redoing help.waterdata)

File: data/roadmap/roadmap.yaml (hunk 3 above)
Comment (srgregory-po) [reply data/roadmap/roadmap.yaml:70]: I'll add some clarification to this entry and the similar one right after that will say "Dynamic site-specific <whatever it is> that can be added by Water Science Center staff"

File: data/roadmap/roadmap.yaml (hunk 4 above)
Comment (srgregory-po) [reply data/roadmap/roadmap.yaml:132]: Added an upcoming thing for that other discovery

File: data/roadmap/roadmap.yaml (hunk 1 above)
Comment (srgregory-po) [reply data/roadmap/roadmap.yaml:60]: We only did alpha code for Search and Filter, but we had alpha-level renderings (paper prototypes) that we used during that time for the download page.  You think this line is misleading?

Comment (chopkins-usgs) [reply data/roadmap/roadmap.yaml:60]: Not misleading, but maybe we could put that alpha phase is Done already

Comment (srgregory-po) [reply data/roadmap/roadmap.yaml:60]: That's fair.  Line added for that one to make it more clear.

**Pull request merged**